libraries/wasm-jit/Source/Runtime/Memory.cpp

```diff
@@ -134,11 +134,10 @@ namespace Runtime
 	
 	U8* getValidatedMemoryOffsetRange(MemoryInstance* memory,Uptr offset,Uptr numBytes)
 	{
-		if(!memory)
-			causeException(Exception::Cause::accessViolation);
 		// Validate that the range [offset..offset+numBytes) is contained by the memory's reserved pages.
 		U8* address = memory->baseAddress + offset;
-		if(	address < memory->reservedBaseAddress
+		if(	!memory
+		||	address < memory->reservedBaseAddress
 		||	address + numBytes < address
```

Comment:
Line 138 uses `memory` before it is checked for null on line 139.

Reply:
Yep, I know, I fixed this previously because our code called through this path. But now that we no longer go through this path, I am changing it back to upstream to make future merges easier & reduce the changeset between us and upstream.

https://github.com/AndrewScheidecker/WAVM/blob/master/Source/Runtime/Memory.cpp#L135